app/experimenter/experiments/views.py

```diff
@@ -118,6 +145,22 @@ class ExperimentFilterset(filters.FilterSet):
     archived = filters.BooleanFilter(
         label="Show archived experiments", widget=forms.CheckboxInput()
     )
+    experiment_date_field = filters.ChoiceFilter(
+        empty_label="No Date Restriction",
+        choices=[
+            ("starts_between", "Experiment Starts"),
```

Comment:
I added them to the `experimenter/experiments/constants.py` file, so that I could call them by saying `Experiment.EXPERIMENT_STARTS` and they are in scope in other places like the filter method. It's done like this in the other filters. Do you dislike this?

Reply:
Yep that seems good!